cpp/models/lct_secir_2_diseases/parameters.h

```diff
@@ -0,0 +1,665 @@
+
+/**
+ * @brief Average time spent in the Exposed compartment for disease a in day unit.
+ */
+struct TimeExposed_a {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TimeExposed_a";
+    }
+};
+
+/**
+ * @brief Average time spent in the TimeInfectedNoSymptoms before developing 
+ *  symptoms or recover for disease a in day unit.
+ */
+struct TimeInfectedNoSymptoms_a {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TimeInfectedNoSymptoms_a";
+    }
+};
+
+/**
+ * @brief Average time spent in the TimeInfectedSymptoms before going to hospital 
+ *  or recover for disease a in day unit.
+ */
+struct TimeInfectedSymptoms_a {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TimeInfectedNoSymptoms_a";
+    }
+};
+
+/**
+ * @brief Average time being in the Hospital before treated by ICU or recover for disease a in day unit.
+ */
+struct TimeInfectedSevere_a {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TimeInfectedSevere_a";
+    }
+};
+
+/**
+ * @brief Average time treated by ICU before dead or recover for disease a in day unit.
+ */
+struct TimeInfectedCritical_a {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TimeInfectedCritical_a";
+    }
+};
+
+/**
+ * @brief Probability of getting infected from a contact for disease a.
+ */
+struct TransmissionProbabilityOnContact_a {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TransmissionProbabilityOnContact_a";
+    }
+};
+
+/**
+ * @brief Average time spent in the Exposed compartment for disease b in day unit.
+ */
+struct TimeExposed_b {
+    using Type = Eigen::VectorX<UncertainValue<ScalarType>>;
+    static Type get_default(size_t size = 1)
+    {
+        return Type::Constant(size, 1, 1.);
+    }
+    static std::string name()
+    {
+        return "TimeExposed_b";
+    }
+};
+
+/**
+ * @brief Average time spent in the TimeInfectedNoSymptoms before developing 
+ *  symptoms or recover for disease b in day unit.
```

Comment:
```suggestion
 *  symptoms or recovering for disease b in day unit.
```
Same for other parameters.